packages/maker.js/src/core/maker.ts

```diff
@@ -782,6 +782,24 @@ namespace MakerJs {
         alternateDirection?: boolean;
     }
 
+    /**
+     * A point reference in a path.
+     * Ratio is not available for toKeyPoints
```

Comment:
Let's remove this comment, and undo the change to `toKeyPoints`

Reply:
done